docs/modules/trino/pages/usage-guide/connect_to_trino.adoc

```diff
@@ -47,6 +53,8 @@ image::connect-with-dbeaver-3.png[]
 
 As the last step you can click on _Finish_ and start using the Trino connection.
 
+TIP: In case you are using OpenID connect, set the `externalAuthentication` property to `true` and don't provide and username or password. A browser window will be opened, which might require you to log in.
```

Comment:
Did this recently change? I recall the `--user` flag being required even with external auth (else $USER is used), so I assume that got fixed. If so, can we link to the issue for posterity?

Reply:
Good catch, this is still an issue! Should be fixed in 8027f64